### DISCUSSION TITLE
Dúvida sobre Publish no Nuget

### DISCUSSION THREAD

**Discussion by JaqueManzano** [balta-io/forum#455] [Tira Dúvidas] [answered]
Balta, estou tentando realizar o publish  do meu projeto MoneyExtension conforme o Curso **2812 - Fundamentos do Azure, Git, Github e DevOps .. porém após realizar todo o processo, criar o yml e enviar ao github estou enfrente a mensagem conforme print, todos os passos exceto o Publish funcionam corretamente.
![image](https://github.com/balta-io/forum/assets/143463393/c7fa758d-95c7-4f0a-ad68-a4fe9a9b8105)

Já tentei excluir o token e criar novamente no nuget, já adicionei o nome do pacote no yml no caminho do dotnet nuget push, já tentei fazer o publish colocando apenas o *.nupkg

![image](https://github.com/balta-io/forum/assets/143463393/7a175029-cea1-4489-b7dc-7257d03f395a)

No yml coloquei o nome da branch como master pois o meu repositório encontra-se com a branch master e não main.

![image](https://github.com/balta-io/forum/assets/143463393/ab4ebbc5-5ef9-4e4b-813f-99f696b24fec)

Com essas informações que lhe enviei, você possui alguma sugestão do que posso alterar para conseguir fazer o publish?
Vi uma discussão semelhante que você comenta sobre o PAT, vi como criar na internet porém na entendi como deve ser usado o mesmo e se isso resolveria o problema, você informa que comenta sobre o PAT nos vídeos porém não encontrei, caso seja esse o possível problema, poderia por gentileza me enviar o link para visualização de sua aula referente ao PAT?

Obrigada!

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @JaqueManzano, como vai?
Provavelmente é por conta do nome do pacote está diferente do repositório:
👉 https://github.com/balta-io/forum/discussions/226#discussioncomment-5575871 

Veja se te ajuda e retorna aqui caso não funcione.

**Reply by JaqueManzano** [reply]
Perfeito @andrebaltieri , era isso mesmo. Muito obrigada!

**Reply by andrebaltieri** [reply]
Show 🚀

**Comment by JaqueManzano** [comment]
Conforme sugerido pelo Balta, ajustei para o nome do pacote do token ficar igual ao nome do package Id e também renomeei o pacote pois já existia pacote chamado MoneyExtensions.
